searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -435,6 +447,13 @@ searxng.ready(function () {
     }
   }
 
+  function copyURLToClipboard () {
+    var current = document.querySelector('.result[data-vim-selected] h3 a');
+    if (current === null) return;
+    var url = current.getAttribute('href');
+    navigator.clipboard.writeText(url);
+  }
+
```

Comment:
```suggestion
  function copyURLToClipboard() {
    const currentUrlElement = document.querySelector('.result[data-vim-selected] h3 a');
    if (currentUrlElement === null) return;
    
    const url = currentUrlElement.getAttribute('href');
    navigator.clipboard.writeText(url);
  }

```

Reply:
I did use `function name ()` as in same file all functions are declared as such including the `toggleHelp` which too takes no other argument so that my changes are consistent with the other code. I am not commiting this part, but will surely rename the variable. Let me know if you want this change to happen.